shared/js/loadApp.js

```diff
@@ -29,7 +29,7 @@ module.exports = function (callback) {
     }
   };
 
-  if (!appOptions.channel) {
+  if (!appOptions.channel && dashboard.clientState.queryParams('solution') !== 'true') {
```

Comment:
Descriptive local variable would work too; I guess a function might be overkill.